static/js/src/logicmodel/table.js

```diff
@@ -66,6 +66,7 @@ LogicModel.TableView = Backbone.View.extend({
         var self = this;
```

Comment:
cool - I've been using `me` for this purpose but I like `self` more.. more Python-like.